### ISSUE TITLE
RVC Mode clusters: Remove Idel constraints.

### ISSUE THREAD

**Issue by hicklin** [project-chip/connectedhomeip#33591]
Add the `DIRECTMODECH` feature to the RvcRunMode and RvcCleanMode clusters as described in [this PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9221).

This feature disables the constraint that disallows a mode change between two modes that do not have an idle mode tag

**Comment by kiel-apple** [comment]
@hicklin does this still seem doable?

**Comment by hicklin** [comment]
No. TBH, given the amount of change I'm not sure what's doable anymore. But currently this is at the bottom of the priority list so unless the priority changes, this is unlikely to happen for this release.
